arguments/golang.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	DefaultGoVersion      = "1.22.3"
+	DefaultGoVersion      = "1.22.4"
```

Comment:
1.22.5 if you want to be current 😅

Reply:
we should probably update that in grafana huh :eyes: